app/src/main/java/org/akvo/flow/dao/SurveyDbAdapter.java

```diff
@@ -171,11 +176,12 @@
     }
 
     public interface SurveyInstanceStatus {
-        int SAVED      = 0;
-        int SUBMITTED  = 1;
-        int EXPORTED   = 2;
-        int SYNCED     = 3;
-        int DOWNLOADED = 4;
+        int UNKNOWN = 0;
```

Comment:
How will changing these affect any data that is already in the app when the update happens?